addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -317,14 +317,29 @@ function registerWebsitePreviewTour(name, options, steps) {
     return tour.register(name, Object.assign({}, options, { url }), tourSteps);
 }
 
-function registerThemeHomepageTour(name, steps) {
+function registerThemeHomepageTour(name, steps, cssVariableChecks) {
+    const preconditionSteps = [];
+    if (cssVariableChecks) {
+        for (const cssVariable of Object.keys(cssVariableChecks)) {
+            preconditionSteps.push({
+                content: `Check CSS variable ${cssVariable}=${cssVariableChecks[cssVariable]}`,
+                trigger: 'iframe body',
+                run: function () {
+                    const style = getComputedStyle(this.$anchor[0]);
+                    if (style.getPropertyValue(cssVariable).trim() !== cssVariableChecks[cssVariable].trim()) {
+                        throw new Error(`Failed precondition: ${cssVariable}=${style.getPropertyValue(cssVariable)} (should be ${cssVariableChecks[cssVariable]})`);
+                    }
+                },
```

Comment:
Why don't we just make this a step util that receive a css variable, and make the theme tour add a new step `wTourUtils.checkCssVariables({..})`?
- It will not "noise" this method with this 'unrelated' param
- It will also have the advantage that it will be usable also in regular tests if we need it

Reply:
It was to be able to make that parameter mandatory - but indeed, having a new kind of step for this will make it more reusable.